.azure/pipelines/stages/publish-symbols.yml

```diff
@@ -0,0 +1,67 @@
+# publish-symbols.yml
```

Comment:
Once we have eng/common/CheckSymbols.ps1, adding a step to confirm the uploads succeeded will be trivial.

Reply:
It's there now. Just merged https://github.com/aspnet/AspNetCore/pull/10405